CleverTap/Runtime/Native/UnityNativeWrapper/Models/UnityNativeAppInboxPersistence.cs

```diff
@@ -0,0 +1,300 @@
+using UnityEngine;
```

Comment:
Add preprocessor for Unity Native/Editor
`#if (!UNITY_IOS && !UNITY_ANDROID) || UNITY_EDITOR`